lib/middleware/__test__/notificationsMiddleware.test.ts

```diff
@@ -0,0 +1,9 @@
+const { notifyAboutUpdates } = require('../notificationsMiddleware');
+
+describe('lib/middleware/notificationsMiddleware', () => {
+  describe('notifyAboutUpdates()', () => {
+    it('should safely execute without throwing an error', () => {
+      expect(() => notifyAboutUpdates()).not.toThrow();
```

Comment:
This was a PITA to test b/c it's commonJS and also exports a single default export that's a class that you instantiate. So I just made sure it's not throwing. I'm considering removing update-notifier as a dep soon anyways so I think this is fine for now.